src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -69,43 +70,51 @@
+  (let [token-symbol           (:token token)
+        token-data             (first (rf/sub [:wallet/current-viewing-account-tokens-filtered
+                                               token-symbol]))
+        fiat-unformatted-value (get-in token [:values :fiat-unformatted-value])
+        has-balance?           (money/greater-than fiat-unformatted-value (money/bignumber "0"))
+        selected-account?      (rf/sub [:wallet/current-viewing-account-address])
+        token-owners           (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
+        send-params            (if selected-account?
+                                 {:token       token-data
+                                  :stack-id    :screen/wallet.accounts
+                                  :start-flow? true
+                                  :owners      token-owners}
+                                 {:token-symbol token-symbol
+                                  :stack-id     :wallet-stack
+                                  :start-flow?  true
+                                  :owners       token-owners})
+        bridge-params          (if selected-account?
+                                 {:token            token-data
+                                  :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
+                                  :stack-id         :screen/wallet.accounts
+                                  :start-flow?      true
+                                  :owners           token-owners}
+                                 {:token-symbol     token-symbol
+                                  :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
+                                  :stack-id         :wallet-stack
+                                  :start-flow?      true
+                                  :owners           token-owners})]
     [quo/action-drawer
-     [(cond->> [(when (ff/enabled? ::ff/wallet.assets-modal-manage-tokens)
-                  (action-manage-tokens watch-only?))
-                (when (ff/enabled? ::ff/wallet.assets-modal-hide)
-                  (action-hide))]
-        (not watch-only?) (concat [(action-buy)
-                                   (when (seq token-owners)
-                                     (action-send send-params entry-point))
-                                   (action-receive selected-account?)
-                                   (when (ff/enabled? ::ff/wallet.swap) (action-swap))
-                                   (when (seq (seq token-owners))
-                                     (action-bridge bridge-params))]))]]))
+     (let [show-send?   (if selected-account?
+                          (and (seq token-owners) has-balance?)
+                          (seq token-owners))
+           show-bridge? (if selected-account?
+                          (and (seq (seq token-owners)) has-balance?)
+                          (seq (seq token-owners)))]
```

Comment:
are the double seq intentional `(seq (seq ...))`?

Reply:
that was there before so I won't change that for now.